src/main/java/com/khutircraftubackend/seller/SellerResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.khutircraftubackend.seller;
+
+import lombok.Builder;
+
+import java.time.LocalDateTime;
+
+@Builder
+public record SellerResponse(
```

Comment:
do we still need SellerDTO and SellerMapper ?

Reply:
There may not be much need now, but when adding methods, 90% of what will be needed